test/spec/provider/zeebe/Forms.spec.js

```diff
@@ -44,7 +44,7 @@ describe('provider/zeebe - Forms', function() {
     BpmnPropertiesPanel,
     CoreModule,
     ModelingModule,
-    SelectionModule,
+    SelectEntryionModule,
```

Comment:
I think we renamed too many modules here 😅

Reply:
Indeed!

Reply:
Fixed via https://github.com/bpmn-io/bpmn-js-properties-panel/pull/525/commits/abef6050fdafd92c8a7e73381a62f79f27de4d12